src/http_rest_api_handler.cpp

```diff
@@ -411,24 +268,28 @@ static bool isInputEmpty(const ::KFSRequest::InferInputTensor& input) {
     return true;
 }
 
-static Status handleBinaryInput(const int binary_input_size, size_t& binary_input_offset, const size_t binary_buffer_size, const char* binary_inputs, ::KFSRequest::InferInputTensor& input) {
+static Status validateContentFieldsEmptiness(KFSTensorInputProto& input) {
+    if (!isInputEmpty(input)) {
+        SPDLOG_DEBUG("{} contents is not empty. Content field should be empty when using binary inputs extension.", input.datatype());
```

Comment:
Binary inputs extension is related to REST, but here we check emptiness inside gRPC proto. Isn't the log message misleading? Should state that contents must be empty when data is placed in raw field?